assets/_scss/components/_accordions.scss

```diff
@@ -0,0 +1,46 @@
+#accordion-menu {
+  border-color: $color-grey-600;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom-width: 0;
+  color: $color-grey-900;
+  margin: 0;
+  padding: 0;
+
+  li {
```

Comment:
These selectors are fragile, in that if somebody wants to make an accordion out of different elements then ps for headers, the code wouldn't work. I suggest using classes instead to ensure the code breaks in less situations.